### PR TITLE
chore: add Readme.md links in each Cargo.toml

### PR DESCRIPTION
**Pull request by 15IITian**
Added Readme.md links in those Crate.toml where there exists a README.md file in that individual crate 
#3924

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -61,6 +61,7 @@ keywords = ["bitcoin", "lightning", "chaumian", "e-cash", "federated"]
 threshold_crypto = { version = "0.1", package = "fedimint-threshold-crypto" }
 tonic_lnd = { version = "0.1.3", package="fedimint-tonic-lnd", features = ["lightningrpc", "routerrpc"] }
 cln-rpc = { package = "fedimint-cln-rpc", version = "0.4.0" }
+fedimint-aead = { version = "0.0.1" }
```
Comment (dpc) [inline Cargo.toml:64]: ?

Comment (15IITian) [reply Cargo.toml]: By Mistake, I thought that I have to include this line in root Cargo.toml file as per the screenshot :)

**Comment by 15IITian** [comment]
image

Not getting why these links are dead as I have not made any changes in it & they are working fine in the Readme.md file .

**Review by elsirion — APPROVED** [review]
Doesn't fully solve https://github.com/fedimint/fedimint/issues/3924 since crates without README still don't have any linked, but it's an improvement.

**Comment by 15IITian** [comment]
> Doesn't fully solve #3924 since crates without README still don't have any linked, but it's an improvement.

For the crates not having their own README -> I have to give link of the root Readme file?

**Comment by elsirion** [comment]
> For the crates not having their own README -> I have to give link of the root Readme file?

I think that's a complicated problem since the root README isn't included when publishing sub-crates by default (see my comment in the issue). So this requires some Cargo wizardry, which is probably not worth it rn.

**Comment by 15IITian** [comment]
> > For the crates not having their own README -> I have to give link of the root Readme file?
> 
> I think that's a complicated problem since the root README isn't included when publishing sub-crates by default (see my comment in the issue). So this requires some Cargo wizardry, which is probably not worth it rn.

Okk,
I think that's the same problem in the License field case in each Cargo.toml?

**Comment by 15IITian** [comment]
@elsirion
 For testing,  I have published one sub_crate  named `another_trial`  from  a workspace , which do not has its own README file ->  thus has relative path of the root README file-> and that's working fine as required ->
 
image

image

**Review by dpc — APPROVED** [review]

**Comment by elsirion** [comment]
Interesting, then let's try this in another PR!

**Pull request merged**